cmd/ak/cmd/up.go

```diff
@@ -13,6 +16,8 @@ import (
 
 var mode string
 
+var App *fx.App
```

Comment:
Making `fx.App` a global and exported variable creates a huge risk. Now any code anywhere in the server can mess with it directly and damage the server, intentionally or not. Even if there's no damage, this goes against the separation of concerns principle.